Cargo.toml

```diff
@@ -11,11 +11,16 @@ Library to support the reading and writing of zip files.
 edition = "2018"
 
 [dependencies]
-flate2 = { version = "1.0.0", default-features = false, optional = true }
-time = { version = "0.3", features = ["formatting", "macros" ], optional = true }
+aes = { version = "0.6.0", optional = true }
```

Comment:
What about using the latest aes version [0.7.5](https://crates.io/crates/aes)?
There seems to be breaking changes in the API.

Thank you for the hard work!

Reply:
I tried updating aes, pbkdf2, hmac and sha-1 (now sha1) together since they all use the same dependencies. Since this required some small changes to the decryption code, I also put in some more (possibly meaningless) checks.

This increased the performance of AES-128 from 42 MB/s to around 160 - 180 MB/s.